app/views/users/edit.html.erb

```diff
@@ -4,42 +4,54 @@
 
   <div class="settingsWrapper">
     <div class="settingsSection">
-      <% unless @user.google_id %>
-        <p>If you'd like to sign in to Volition with your Google account, click below.</p>
-        <%= render partial: "sessions/google_sign_in" %>
-        <%= hidden_field_tag :google_id, id: 'google_id_token' %>
+      <% unless self_hosted? %>
+        <% unless @user.google_id %>
```

Comment:
We can do something like this: `unless @user.google_id || self_hosted?`.

Allows us to not nest conditionals as much.